### DISCUSSION TITLE
Windows nightly build

### DISCUSSION THREAD

**Discussion by giuliovale** [rustdesk/rustdesk#1839] [General]
Hello, but in the night build there are not all the functions such as screen shrink, monitor change, etc?
Thanks

**Comment by rustdesk** [comment]
there are. Please test and feed back. @fufesou . It seems guys are not easy to find the toolbar. We need to add another tool icon next to settings icon on the titlebar rightside.

**Comment by rustdesk** [comment]
Or show and stick the toolbar by default, remember the show/ stick status, next time restore it.

**Comment by giuliovale** [comment]
This is the screenshot, I not see any toolbar

![desk](https://user-images.githubusercontent.com/6909743/198205260-04d448c0-f846-403c-a99b-d4aa5767e5e5.jpg)

**Comment by mikekgr** [comment]
I have exactly the same problem. Not possible to activate or get these screen etc functions.

**Comment by rustdesk** [comment]
![image](https://user-images.githubusercontent.com/71636191/198207748-f045b52b-8096-46a1-8039-1e8af76c0197.png)
We do need to redesign to make users easy to find it.

**Comment by giuliovale** [comment]
Uauu there is, but for sure the color need to be more evident
Thanks

**Comment by mikekgr** [comment]
I found it now. Needs to be emphasized better...
